package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hubs-discord-bot",
-  "version": "0.0.2",
+  "version": "0.0.5",
```

Comment:
Same here, I wouldn't bother changing these versions. It doesn't affect anything.